pkg/loaders/api.go

```diff
@@ -0,0 +1,11 @@
+package loaders
+
+import (
+	"go-calendar-practice/pkg/modules/example"
+
+	"github.com/gin-gonic/gin"
+)
+
+func LoadApis(r *gin.Engine) {
```

Comment:
```suggestion
func LoadAPIs(r *gin.Engine) {
```
Go에서 약어를 사용하는 경우에는 약어 문자 전체를 UPPERCASE 혹은 lowercase 로 적어주세요. 일반적으로 Api, Url, Http 같은 방식은 Go에서 사용되지 않습니다

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

[Reference](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
알겠습니다! 레퍼런스까지 남겨주셔서 감사합니다